docs/v0.7/en/release-notes/release-0-7-0.md

```diff
@@ -0,0 +1,151 @@
+* feat: enable zstd compression and encodings in merge tree data part by [@v0y4g3r](https://github.com/v0y4g3r) in [#3380](https://github.com/GreptimeTeam/greptimedb/pull/3380)
+* feat(flow): impl for MapFilterProject by [@discord9](https://github.com/discord9) in [#3359](https://github.com/GreptimeTeam/greptimedb/pull/3359)
+* feat: flush or compact table and region functions by [@killme2008](https://github.com/killme2008) in [#3363](https://github.com/GreptimeTeam/greptimedb/pull/3363)
+* feat: Use a partition level map to look up pk index by [@evenyag](https://github.com/evenyag) in [#3400](https://github.com/GreptimeTeam/greptimedb/pull/3400)
+* feat(index): measure memory usage in global instead of single-column and add metrics by [@zhongzc](https://github.com/zhongzc) in [#3383](https://github.com/GreptimeTeam/greptimedb/pull/3383)
+* feat: enable ArrowFlight compression by [@tisonkun](https://github.com/tisonkun) in [#3403](https://github.com/GreptimeTeam/greptimedb/pull/3403)
+* feat: zero copy on split rows by [@fengjiachun](https://github.com/fengjiachun) in [#3407](https://github.com/GreptimeTeam/greptimedb/pull/3407)
+* feat: Support automatic DNS lookup for kafka bootstrap servers by [@J0HN50N133](https://github.com/J0HN50N133) in [#3379](https://github.com/GreptimeTeam/greptimedb/pull/3379)
+* feat: add configuration for tls watch option by [@sunng87](https://github.com/sunng87) in [#3395](https://github.com/GreptimeTeam/greptimedb/pull/3395)
+* feat: employ sparse key encoding for shard lookup by [@v0y4g3r](https://github.com/v0y4g3r) in [#3410](https://github.com/GreptimeTeam/greptimedb/pull/3410)
+* feat: support `Create Table ... Like` by [@KKould](https://github.com/KKould) in [#3372](https://github.com/GreptimeTeam/greptimedb/pull/3372)
+* feat: tableref cache by [@fengjiachun](https://github.com/fengjiachun) in [#3420](https://github.com/GreptimeTeam/greptimedb/pull/3420)
+* feat: add verbose support for tql explain/analyze by [@etolbakov](https://github.com/etolbakov) in [#3390](https://github.com/GreptimeTeam/greptimedb/pull/3390)
+* feat: reduce a clone of string by [@fengjiachun](https://github.com/fengjiachun) in [#3422](https://github.com/GreptimeTeam/greptimedb/pull/3422)
+* feat: Correct server metrics and add more metrics for scan by [@evenyag](https://github.com/evenyag) in [#3426](https://github.com/GreptimeTeam/greptimedb/pull/3426)
+* feat: support tracing rule sampler by [@Taylor-lagrange](https://github.com/Taylor-lagrange) in [#3405](https://github.com/GreptimeTeam/greptimedb/pull/3405)
+* feat: decode prom requests to grpc by [@v0y4g3r](https://github.com/v0y4g3r) in [#3425](https://github.com/GreptimeTeam/greptimedb/pull/3425)
+* feat: implement multi-dim partition rule by [@waynexia](https://github.com/waynexia) in [#3409](https://github.com/GreptimeTeam/greptimedb/pull/3409)
+
```

Comment:
Is `New Contributors` no longer included?

Reply:
I choose All Contributors by `git log --pretty="%an%n%cn" v0.6.0..v0.7.0 | sort | uniq` and manually edit.

A dedicated section for new contributors driven by GitHub can be desired.